config/routes.rb

```diff
@@ -6,4 +6,7 @@
 
   # Uncomment this line to activate the endpoint for the Square hosted checkout workflow.
   # post 'square_checkout', to: '/solidus_square/callback_actions#square_checkout'
+
+  # Uncomment this line to activate the endpoint for the Square order.updated Webhook
+  # patch "webhooks/square", to: '/solidus_square/webhooks#update'
```

Comment:
This should be a POST

Reply:
🤔 Maybe I should change the method from update to create? @DanielePalombo

Reply:
yes, it makes sense